x-pack/plugins/ml/public/application/data_frame_analytics/pages/analytics_management/components/action_clone/clone_action_name.tsx

```diff
@@ -363,6 +363,14 @@ export const useNavigateToWizardWithClonedJob = () => {
       );
       if (ip !== undefined) {
         sourceIndexId = ip.id;
+      } else {
+        toasts.addDanger(
+          i18n.translate('xpack.ml.dataframe.analyticsList.noSourceIndexPatternForClone', {
+            defaultMessage:
+              'Unable to clone the analytics job. No index pattern exists for index {indexPattern}.',
```

Comment:
Is it possible to add a link to the Index Patterns management page in this toast?

Reply:
The notification service doesn't accept elements currently. I think it will be okay to leave as is with the error message - happy to follow up with switching to using a modal or something if it's something people notice they want.